tutorial-content/content/weighting-schemes.md

```diff
@@ -279,6 +323,17 @@ After increasing the resolution of the LandScan data, we clip it again.
 landscan <- crop(landscan, extent(-125.0208, -66.47917, 24.0625, 49.9375))
 ```
 ````
+````{tab-item} Python
+```Python
+xmin, xmax, ymin, ymax = -125.0208, -66.47917, 24.0625, 49.9375
+
+# Calculate the window to crop
+window = from_bounds(xmin, ymin, xmax, ymax, landscan.transform)
+
+# Extract data from this window
+landscan_cropped_data = landscan.read(1, window=window)
```

Comment:
You keep transforming landscan, but setting the result as a new variable. To apply all of the operations, it's probably best to set the result back into the landscan variable, like we do in R.

Reply:
Ok!